CHANGELOG.md

```diff
@@ -974,6 +980,7 @@ code.
 - `avifVersion()` function
 
 [Unreleased]: https://github.com/AOMediaCodec/libavif/compare/v1.0.0...HEAD
+[1.0.1]: https://github.com/AOMediaCodec/libavif/compare/v1.0.0...v1.0.1
```

Comment:
Note that I left the `[Unreleased]` link unchanged because v1.0.0 is the last release on the main branch.